recipes/icu/run_test.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+set -e
+
+icu-config --help
```

Comment:
Putting this in `meta.yaml` didn't seem work for me at the time. Maybe the `-` confused it. Though there were other problems that made it difficult to tell if this itself was having issues or not. So maybe we could try to add it back to `meta.yaml` and see if it works. If not, it can just go back here.